run-tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+# Install testing libs and run tests.
+pip install \
+    pytest==3.1.1 \
+    pytest-cover==3.0.0 \
+    pytest-flake8==0.6 \
+    pyflakes==1.2.3 \
+    flake8==3.0.0
+
+# If RUN_INTEGRATION_TESTS is set, run integration tests.
+echo "Run integration tests? ${RUN_INTEGRATION_TESTS}"
+if [ "$RUN_INTEGRATION_TESTS" = "True" ] ; \
+    then echo "Run all tests" && \
+        python -mpy.test \
```

Comment:
typo

Reply:
For what it's worth, it's not uncommon to use Python's `-m` argument without a space between the `-m` and the module being called.  See the example from Matplotlib: https://matplotlib.org/faq/installing_faq.html (look for `python -mpip install matplotlib`)

But it works either way.